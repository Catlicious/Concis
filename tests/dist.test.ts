describe('test web-react pkg', () => {
  it('test pc dist cjs module correctly', () => {
    const concis = require('../packages/concis-react/web-react/cjs');
    expect(Object.keys(concis)).toMatchSnapshot();
  });

  it('test pc dist esm module correctly', () => {
    const concis = require('../packages/concis-react/web-react/es');
    expect(Object.keys(concis)).toMatchSnapshot();
  });

  it('test pc dist umd module correctly', () => {
    const concis = require('../packages/concis-react/web-react/umd');
    expect(Object.keys(concis)).toMatchSnapshot();
  });
});

export {};
