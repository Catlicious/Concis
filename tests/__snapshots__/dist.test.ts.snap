// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test web-react pkg test pc dist cjs module correctly 1`] = `
Array [
  "Affix",
  "Alert",
  "Avatar",
  "AvatarGroup",
  "BackTop",
  "Badge",
  "Breadcrumb",
  "Button",
  "CheckBox",
  "Collapse",
  "CollapseItem",
  "Comment",
  "Content",
  "DatePicker",
  "Divider",
  "Drawer",
  "Dropdown",
  "Empty",
  "Footer",
  "Form",
  "GlobalConfig",
  "Header",
  "Image",
  "Input",
  "InputPro",
  "Layout",
  "LazyLoad",
  "Link",
  "List",
  "Loading",
  "Menu",
  "Message",
  "Modal",
  "MonthPicker",
  "Nav",
  "Notification",
  "Pagination",
  "Popover",
  "PreviewJump",
  "Radio",
  "RadioGroup",
  "RangeDatePicker",
  "Rate",
  "Select",
  "Skeleton",
  "Slider",
  "Space",
  "Step",
  "Steps",
  "Swiper",
  "Switch",
  "Table",
  "Tabs",
  "Tag",
  "TextArea",
  "TimePicker",
  "Tree",
  "TreeView",
  "Upload",
  "YearPicker",
]
`;

exports[`test web-react pkg test pc dist esm module correctly 1`] = `
Array [
  "Affix",
  "Alert",
  "Avatar",
  "AvatarGroup",
  "BackTop",
  "Badge",
  "Breadcrumb",
  "Button",
  "CheckBox",
  "Collapse",
  "CollapseItem",
  "Comment",
  "Content",
  "DatePicker",
  "Divider",
  "Drawer",
  "Dropdown",
  "Empty",
  "Footer",
  "Form",
  "GlobalConfig",
  "Header",
  "Image",
  "Input",
  "InputPro",
  "Layout",
  "LazyLoad",
  "Link",
  "List",
  "Loading",
  "Menu",
  "Message",
  "Modal",
  "MonthPicker",
  "Nav",
  "Notification",
  "Pagination",
  "Popover",
  "PreviewJump",
  "Radio",
  "RadioGroup",
  "RangeDatePicker",
  "Rate",
  "Select",
  "Skeleton",
  "Slider",
  "Space",
  "Step",
  "Steps",
  "Swiper",
  "Switch",
  "Table",
  "Tabs",
  "Tag",
  "TextArea",
  "TimePicker",
  "Tree",
  "TreeView",
  "Upload",
  "YearPicker",
]
`;

exports[`test web-react pkg test pc dist umd module correctly 1`] = `
Array [
  "Affix",
  "Alert",
  "Avatar",
  "AvatarGroup",
  "BackTop",
  "Badge",
  "Breadcrumb",
  "Button",
  "CheckBox",
  "Collapse",
  "CollapseItem",
  "Comment",
  "Content",
  "DatePicker",
  "Divider",
  "Drawer",
  "Dropdown",
  "Empty",
  "Footer",
  "Form",
  "GlobalConfig",
  "Header",
  "Image",
  "Input",
  "InputPro",
  "Layout",
  "LazyLoad",
  "Link",
  "List",
  "Loading",
  "Menu",
  "Message",
  "Modal",
  "MonthPicker",
  "Nav",
  "Notification",
  "Pagination",
  "Popover",
  "PreviewJump",
  "Radio",
  "RadioGroup",
  "RangeDatePicker",
  "Rate",
  "Select",
  "Skeleton",
  "Slider",
  "Space",
  "Step",
  "Steps",
  "Swiper",
  "Switch",
  "Table",
  "Tabs",
  "Tag",
  "TextArea",
  "TimePicker",
  "Tree",
  "TreeView",
  "Upload",
  "YearPicker",
]
`;
